src/__docker_tests__/quickTest.ts

```diff
@@ -84,8 +75,13 @@ const waitForPredicate = <T1>(successResult: T1, predicate: (a: T1) => boolean,
             if (predicate(successResult)) {
                 resolve();
             } else {
-                reject();
+                reject("Predicate not satisfied.");
             }
         }, timeout);
     });
 };
+
+console.log("Executing smoke tests");
+setup().then(() => {
+    execute().then(() => console.log("Tests passed!"));
+});
```

Comment:
this approach to test works, but i dont think its scalable, is there any particular reason for not using mocha? or is it just an optimisation

Reply:
I agree. I removed mocha temporarily to reduce the dependencies required to run tests - this was initially because tests were part of the deployment. I've created a new 'test' docker image to contain these test files, and it will contain the relevant environment - mocha - to run integration tests. So we'll migrate back to mocha on a later branch.